using-mobile-extensions/adobe-analytics/README.md

```diff
@@ -18,7 +18,7 @@
 Want to know how to find report suite IDs? See [Report Suites](https://docs.adobe.com/content/help/en/analytics/admin/manage-report-suites/report-suites-admin.html).
 {% endhint %}
 
-Type one or more report suite identifiers to which the Analytics data should be sent. To add multiple report suite IDs, click **Add Another**; to remove these IDs, click **Remove Circle.** Report suite IDs can also be configured for the Development and Staging environments.
+Select one or more report suite identifiers from the combo box to which the Analytics data should be sent.  To remove a report suite ID, click on the **X** next to the ID to be removed.  Report suite IDs can also be configured for the Development and Staging environments.
```

Comment:
please reword the end of the 2nd sentence to read as follows:

"...click on the X next to the ID that you want to remove."